test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -211,9 +211,8 @@ def run_data_service_test(ec2_connection, ec2_instance_ami, tensorflow_training,
     python_invoker = get_python_invoker(ec2_instance_ami)
     _, tensorflow_version = test_utils.get_framework_and_version_from_tag(tensorflow_training)
     ec2_connection.run(f"{python_invoker} -m pip install --upgrade pip")
-    ec2_connection.run(f"{python_invoker} -m pip install 'protobuf>=3.20,<3.21'")
     ec2_connection.run(f"{python_invoker} -m pip install tensorflow=={tensorflow_version}")
-    ec2_connection.run(f"{python_invoker} -m pip install 'protobuf>=3.20,<3.21'")
+    ec2_connection.run(f"{python_invoker} -m pip install 'protobuf>=3.9.2,<3.20'")
```

Comment:
https://github.com/tensorflow/tensorflow/blob/b59935d3b349e752eed4729606e5d64a17e48015/tensorflow/tools/pip_package/setup.py#L105

Protobuf downgraded. to avoid issues seen in previous iterations

Reply:
what issues are seen?

Reply:
protoc issues were seen in UL20.04 in latest 4.X, which require 3.19 to fix- https://github.com/protocolbuffers/protobuf/issues/9954

Reply:
lets add this as a comment in the dockerfile for context

Reply:
Ack